### ISSUE TITLE
Recognize `aria-label` and `aria-labelledby` attributes

### ISSUE THREAD

**Issue by waiting-for-dev** [ginader/HTML5-placeholder-polyfill#27]
For me it's ok that it is needed to use a `label`, as discussed in issue #2, but then the aria attributes `aria-label` and `aria-labelledby` should be recognized, and it is not the case. I need to use this polyfill with an input that has no `label` tag but it does have an `aria-labelledby` attribute, as it is accepted by the WAI-ARIA.

Thanks a lot!
